lib/language_pack/rails2.rb

```diff
@@ -76,6 +81,14 @@ def best_practice_warnings
 
 private
 
+  def has_worker_rake_task?
```

Comment:
I couldn't find a reliable way to get the rake tasks through Ruby directly, though there probably is a way. Let me know if I'm missing something here!